src/EditorFeatures/Test/MoveType/AbstractMoveTypeTestsBase.cs

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.UnitTests;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.MoveType
+{
+    public abstract class AbstractMoveTypeTestsBase : AbstractCodeActionTest
+    {
+        private const string SpanMarker = "[||]";
+        private const string RenameFileCodeActionTitle = "Rename File";
+        private const string RenameTypeCodeActionTitle = "Rename Type";
+
+        private string StripSpanMarkers(string text)
```

Comment:
Why do you need this function?  Why not just use MarkupTestFile to parse the text?